crates/typst-library/src/visualize/gradient.rs

```diff
@@ -549,7 +549,7 @@ impl Gradient {
     }
 
     /// Repeats this gradient a given number of times, optionally mirroring it
-    /// at each repetition.
+    /// at every second repetition.
```

Comment:
```suggestion
    /// every second repetition.
```

This may be more natural, but I'm not a native speaker so I might be wrong.

If this is changed here, it should be changed below as well.